### PR TITLE
brainiac.ms

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

**Comment by adambash** [comment]
https://i.ytimg.com/vi/XPxc9_R4IYk/maxresdefault.jpg

proof of mind controlling abilities

**Comment by adambash** [comment]
https://www.google.com.mx/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwjm78-GsebSAhVOxWMKHdOSAKEQjBwIBA&url=https%3A%2F%2Flh3.googleusercontent.com%2Fproxy%2F_9i4zyDV2xjhljT_9h_Ax25v2wlQyJ1MZAKiioGQCCX0GFqr3vkj0oJnqrnoooDXsqVlYXWawDF_FsMd3-hStVccmJooiMPXEAy8I7pX0NzbDU55ayMBGwiDWyj5EtIrHpZA0OeJvAL9fJ5lDX9HA1ypJrX6N_Y4aZpXzttPMDgw0qz4LKXd61Q1U-Ehrv92r3K6%3Ds0-d&bvm=bv.150120842,d.cGc&psig=AFQjCNHNrF_dDduZkLICXF0T_5o8XqHG_A&ust=1490143949206659

proof of lightning and lasers

**Comment by adambash** [comment]
http://hd.wallpaperswide.com/thumbs/brainiac_dc_comics-t2.jpg

proof of lightning and lasers

**Comment by adambash** [comment]
https://static2.comicvine.com/uploads/original/11125/111253436/5609469-brainuhconvergence-00bnr.jpg

proof of cloning and disguising

**Comment by adambash** [comment]
http://vignette1.wikia.nocookie.net/smallville/images/4/4a/Homecoming_0290.jpg/revision/latest?cb=20101016105921

proof of shapeshifting again in smallville

File: CommandHelper/LocalPackages/commands/buffs/brainiac.ms [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/speak <playername>",
+                "/confuse",
+                "/unconfuse",
+                "/mindlock"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 12 true",
+                "/effect ". @plr ." 11 9999 4 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/brainiac.ms]: requires proof of invincibility

**Pull request merged**